ledger-core/v2/virtual/handlers/vdelegatedcallrequest.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package handlers
+
+import (
+	"context"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor"
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/contract"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/jet"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/v2/log"
+	"github.com/insolar/assured-ledger/ledger-core/v2/network/messagesender"
+	messageSenderAdapter "github.com/insolar/assured-ledger/ledger-core/v2/network/messagesender/adapter"
+	"github.com/insolar/assured-ledger/ledger-core/v2/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/injector"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/throw"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/object"
+)
+
+var deadBeef = [...]byte{0xde, 0xad, 0xbe, 0xef}
+
+type SMVDelegatedCallRequest struct {
+	// input arguments
+	Meta    *payload.Meta
+	Payload *payload.VDelegatedCallRequest
+
+	objectSharedState object.SharedStateAccessor
+
+	// dependencies
+	objectCatalog  object.Catalog
+	messageSender  messageSenderAdapter.MessageSender
+	jetCoordinator jet.AffinityHelper
```

Comment:
I guess it's ok, for now, to use `JetCoordinator` on SM layer, but I insist on one of the following actions in the future:
1. Move `.Me()` function to another place
2. Fill that field of message in other place